interop/xds/client/client.go

```diff
@@ -135,16 +229,52 @@ func main() {
 	}
 	ticker := time.NewTicker(time.Second / time.Duration(*qps**numChannels))
 	defer ticker.Stop()
-	sendRPCs(clients, ticker)
+	sendRPCs(clients, rpcCfgs, ticker)
 }
 
-func sendRPCs(clients []testpb.TestServiceClient, ticker *time.Ticker) {
+func makeOneRPC(c testpb.TestServiceClient, cfg *rpcConfig) (*peer.Peer, *rpcInfo, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), *rpcTimeout)
+	defer cancel()
+	var (
+		p      peer.Peer
+		header metadata.MD
+	)
```

Comment:
if you are going to use a var block, maybe move it down to where `err` is declared and include that too?

Reply:
Done